lib/capistrano/templates/puma.service.erb

```diff
@@ -0,0 +1,40 @@
+# This file tells systemd how to run Puma as a 24/7 long-running daemon.
+#
+# Customize this file based on your bundler location, app directory, etc.
+# Customize and copy this into /usr/lib/systemd/system (CentOS) or /lib/systemd/system (Ubuntu).
+# Then run:
+#   - systemctl enable <%= fetch(:puma_service_unit_name) %>
+#   - systemctl {start,stop,restart} <%= fetch(:puma_service_unit_name) %>
+#
+#
+# Use `journalctl -u <%= fetch(:puma_service_unit_name) %> -rn 100` to view the last 100 lines of log output.
+#
+[Unit]
+Description=Puma HTTP Server for <%= "#{fetch(:application)} (#{fetch(:stage)})" %>
+<%= "Requires=#{fetch(:puma_service_unit_name)}.socket" if fetch(:puma_enable_socket_service) %>
+After=network.target
+
+[Service]
+Type=<%= service_unit_type %>
+WatchdogSec=0
+<%="User=#{puma_user(@role)}" if fetch(:puma_systemctl_user) == :system %>
+WorkingDirectory=<%= current_path %>
+ExecStart=<%= expanded_bundle_command %> exec puma -e <%= fetch(:puma_env) %>
+ExecReload=/bin/kill -USR1 $MAINPID
+<%- Array(fetch(:puma_service_unit_env_files)).each do |file| %>
+<%="EnvironmentFile=#{file}" -%>
+<% end -%>
+<% Array(fetch(:puma_service_unit_env_vars)).each do |environment_variable| %>
+<%="Environment=\"#{environment_variable}\"" -%>
+<% end -%>
+
+# if we crash, restart
+RestartSec=1
+Restart=on-failure
+
+<%="StandardOutput=append:#{fetch(:puma_access_log)}" if fetch(:puma_access_log) %>
+<%="StandardError=append:#{fetch(:puma_error_log)}" if fetch(:puma_error_log) %>
```

Comment:
It looks like the default values for these variables changed in seuros/capistrano-puma@a7d33304c58a8beac9172342a5526efc8a52a5ce, which means that the log files used when using the installer will be different than the ones used after deploying with capistrano :thinking:.

Shall we add these lines to our `deploy.rb` to keep the existing behavior?

```ruby
set :puma_access_log, -> { File.join(shared_path, "log", "puma_access.log") }
set :puma_error_log, -> { File.join(shared_path, "log", "puma_error.log") }
```